data/Music/Mega Man Series/Mega Man 3/Shadow.meta

```diff
@@ -1,3 +1,3 @@
-Mega Man 3 - Shadow Man
+Mega Man III - Shadow Man
```

Comment:
Roman numerals are not appropriate for the classic Mega Man series. Those games use Arabic numerals. There is actually a separate line of Mega Man games on the Game Boy that use Roman numerals, and this change may cause some confusion.

Only use Roman numerals if that is how the game is officially stylized.

```suggestion
Mega Man 3 - Shadow Man
```